library.json

```
@@ -25,8 +25,5 @@
             "test",
             ".github"
         ]
-    },
-    "build": {
-        "flags": "-D CORE_DEBUG_LEVEL=ARDUHAL_LOG_LEVEL_INFO"
     }
-}
+}
```

Always remember to add an empty line at the end of files ([reason](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file)). You can configure VSCode to do that automatically by checking the settings _Files: Insert Final Newline_ and _Files: Trim Final Newlines_.
```suggestion
}

```